File: video/users/client-initialization-rooms/client-initialization.java
// Create a Video Client and connect to Twilio's backend.
VideoClient videoClient = new VideoClient(VideoActivity.this, accessToken);

// Connect to a room
ConnectOptions connectOptions = new ConnectOptions.Builder()
                .roomName("my-room")
                .localMedia(localMedia)
                .build();
Room room = videoClient.connect(connectOptions, roomListener());

File: video/users/client-initialization-rooms/client-initialization.swift
// Create a Video Client and connect to Twilio's backend.
var client = TVIVideoClient(token: accessToken)

// Connect to a room
let connectOptions = TVIConnectOptions { (builder) in
    builder.name = "my-room"
}

if let client = client {
    room = client.connect(with: connectOptions, delegate: self)
}